eazieda/histograms.py

```diff
@@ -0,0 +1,80 @@
+from altair.vegalite.v4.schema.core import Value
+from vega_datasets import data
+import pandas as pd
+import altair as alt
+import numpy as np
+
+
+def histograms(data, features, plot_width=100, plot_height=100, num_cols=2):
+    """
+    Generates histograms for numeric features and bar plots for categorical features
+
+    Parameters
+    ----------
+    data : pandas.core.frame.DataFrame
+        A Pandas Dataframe
+
+    features : list
+        A list of strings that represents feature names
+
+    plot_width: int
+        The width of each features sub plot. Default = 100
+
+    plot_height: int
+        The height of each features sub plot. Default = 100
+
+    num_cols : int
+        The number of columns in the final grid of plots. Default = 2
+
+    Returns
+    -------
+    `altair plot`
+        A combined altair correlation plot
+
+    Examples
+    --------
+    >>> from eazieda.eazieda import histograms
```

Comment:
```suggestion
    >>> from eazieda.histograms import histograms
```

It looks right in the test, I think you might have missed this.